apps/full-site-editing/full-site-editing-plugin/editor-gutenboarding-launch/index.ts

```diff
@@ -71,6 +73,9 @@ function updateEditor() {
 			// Disable href navigation
 			e.preventDefault();
 			await dispatch( 'core/editor' ).savePost();
+
+			recordTracksEvent( 'calypso_editor_launch_click', {} );
```

Comment:
Shouldn't we use here the same `calypso_newsite_` prefix as in Gutenboarding? 
Or maybe prop `flow: 'gutenboarding'` 🤔

Reply:
ah yes, newsite would be good addition, good catch. 👍 

flow seems not needed since we only have this in gutenboarding flow and not elsewhere

Reply:
Updated in https://github.com/Automattic/wp-calypso/pull/43885/commits/f790871b17ad225b0c71b85851a47d4e491e0434